server/routes.js

```diff
@@ -6,14 +6,33 @@
 
 import errors from './components/errors';
 import path from 'path';
+const authMiddlewares=require("./components/auth");
+const redirectUrlFunction = require("./components/auth/redirectUrl");
 
-export default function(app) {
+export default function (app) {
   // Insert routes below
-  app.use('/api/users', require('./api/thing'));
+  app.use('/api/users', require('./api/employee'));
+  //route for logout
+  app.use(authMiddlewares);
```

Comment:
Shouldn't authMiddleware be applicable to all apis. If yes, then why it is placed after user apis